data/126/021/452/3/1260214523.geojson

```diff
@@ -4,6 +4,7 @@
   "properties": {
     "edtf:cessation":"uuuu",
     "edtf:inception":"uuuu",
+    "edtf:superseded":"1915",
```

Comment:
Is **1915** when the place started? We use `edtf:inception` for that instead. Superseded is when it still exists but changed in a significant way (like much larger size or totally different name change).